doc/environment.yml

```diff
@@ -3,13 +3,13 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
-  - python=3.7
+  - python=3.9
   - gcc_linux-64
   - gxx_linux-64
   - numpy
   - scipy
   - six
-  - sphinx>=5.1.0
+  - sphinx>=5.1.0,<6
```

Comment:
I would also add this to https://github.com/pymc-devs/pymc-sphinx-theme, otherwise we'll soon have the same issue in pymc, pymc-examples... (if we aren't already). We use the theme straight from github so the effect will be immediate

Reply:
Done: https://github.com/pymc-devs/pymc-sphinx-theme/commit/973f6ddeb392a56f6bf5e9923b2b4354c3660c62